modules/doc/content/training/index.md

```diff
@@ -1,10 +1,13 @@
 # Training
 
 !style! halign=left
-The following links detail any MOOSE related available training on the horizon.
+Upcoming MOOSE related training:
 !style-end!
 
-
+- George Tech (2023 Sept 12-14)
```

Comment:
Who dis

Reply:
@rpodgorney

Reply:
Full name? 
[Georgia Institute of Technology](https://www.gatech.edu/)

Reply:
this was a joke